neo4j-cli/aura/internal/subcommands/deployment/token/delete.go

```diff
@@ -0,0 +1,68 @@
+package token
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/api"
+	"github.com/spf13/cobra"
+)
+
+func NewDeleteCmd(cfg *clicfg.Config) *cobra.Command {
+	var (
+		organizationId string
+		projectId      string
+		deploymentId   string
+	)
+
+	const (
+		organizationIdFlag = "organization-id"
+		projectIdFlag      = "project-id"
+		deploymentIdFlag   = "deployment-id"
+	)
+
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete the deployment token",
+		Long:  "This endpoint deletes a Fleet Manager deployment token.",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := fmt.Sprintf("/organizations/%s/projects/%s/deployments/%s/token", organizationId, projectId, deploymentId)
+
+			cmd.SilenceUsage = true
+			_, statusCode, err := api.MakeRequest(cfg, path, &api.RequestConfig{
+				Method:  http.MethodDelete,
+				Version: api.AuraApiVersion2,
+			})
+			if err != nil {
+				return err
+			}
+
+			if statusCode == http.StatusNoContent {
+				log.Default().Printf("Deployment token deleted successfully for deployment %s", deploymentId)
```

Comment:
```suggestion
				cmd.Println("Deployment token deleted successfully for deployment %s", deploymentId)
```

Seems more appropriate and consistent with other commands in the codebase. It runs `fmt.Sprintln`on the arguments, so can be swapped directly.